index.html

```diff
@@ -295,358 +321,405 @@ <h3 id="w3c-coordination">W3C Groups</h3>
+          "http://lists.w3.org/Archives/Public/public-tt/">archive</a>) or <a id="public-github" href=
+          "https://github.com/w3c/">GitHub issues</a>. The public is invited to review, discuss and contribute to this
+          work.</p>
+    
+          <p>The Working Group MAY use a Member-confidential mailing list member-tt@w3.org (<a href=
+          "http://lists.w3.org/Archives/Member/member-tt/">archive</a>) for administrative purposes and, at the discretion
+          of the Chairs and members of the group, for member-only discussions in special cases when a participant requests
+          such a discussion.</p>
+      </section>
 
-        <h3 id="external-coordination">External Organizations</h3>
 
-        <p>The Working Group SHOULD, whenever possible, seek interoperability with other timed text formats, API, and
-        applications.</p>
 
-        <p>As such, the Working Group SHOULD seek review with the following external organizations.</p>
+      <section id="decisions">
+        <h2>
+          Decision Policy
+        </h2>
+        <p>
+          This group will seek to make decisions through consensus and due process, per the <a href="https://www.w3.org/Consortium/Process/#Consensus"> W3C Process Document (section 3.3</a>). Typically, an editor or other participant makes an initial proposal, which is then refined in discussion with members of the group and other reviewers, and consensus emerges with little formal voting being required.</p>
+        <p>
+           However, if a decision is necessary for timely progress and consensus is not achieved after careful consideration of the range of views presented, the Chairs may call for a group vote and record a decision along with any objections.
+        </p>
+        <p>
+          To afford asynchronous decisions and organizational deliberation, any resolution (including publication decisions) taken in a face-to-face meeting or teleconference will be considered provisional.
+
+          A call for consensus (CfC) will be issued for all resolutions (for example, via email, GitHub issue or web-based survey), with a response period 10 working days, depending on the chair's evaluation of the group consensus on the issue.
+
+          If no objections are raised by the end of the response period, the resolution will be considered to have consensus as a resolution of the Working Group.
+        </p>
+        <p>
+          All decisions made by the group should be considered resolved unless and until new information becomes available or unless reopened at the discretion of the Chairs or the Director.
+        </p>
+        <p>
+          This charter is written in accordance with the <a href="https://www.w3.org/Consortium/Process/#Votes">W3C Process Document (Section 3.4, Votes)</a> and includes no voting procedures beyond what the Process Document requires.
```

Comment:
not sure... since there is one part to select WG or IG in the template, so the first line could be just a mistake? (feel free to file an issue to the template repository)

https://w3c.github.io/charter-drafts/charter-template.html